app/controllers/article_controller.rb

```diff
@@ -167,6 +167,16 @@ def eds_init
     config.add_sort_field 'oldest', sort: 'oldest', label: 'date (oldest)'
   end
 
+  def fulltext_link
+    _response, document = search_service.fetch(params[:id])
+    url = extract_fulltext_link(document, params[:type])
+    redirect_to url if url.present?
+  rescue => e
+    Honeybadger.notify(e) if defined? Honeybadger
+    flash[:error] = "Sorry, the PDF download was not successful.\nAlternative route: click the title of the article you want, then try the PDF link on the detail page."
```

Comment:
Grrr. Thanks for catching that. 

<img width="686" alt="screen shot 2017-08-11 at 3 53 11 pm" src="https://user-images.githubusercontent.com/1861171/29234682-a0839c84-7ead-11e7-81dd-a1c44544936b.png">